internal/controller/ols_app_server_assets.go

```diff
@@ -398,6 +414,26 @@ func (r *OLSConfigReconciler) getAdditionalCAFileNames(cr *olsv1alpha1.OLSConfig
 
 }
 
+func (r *OLSConfigReconciler) validateCertificateInConfigMap(cmName, fileName string) error {
+
+	cm := &corev1.ConfigMap{}
+	err := r.Get(context.TODO(), client.ObjectKey{Name: cmName, Namespace: r.Options.Namespace}, cm)
+	if err != nil {
+		return fmt.Errorf("failed to get certificate configmap %s/%s: %v", r.Options.Namespace, cmName, err)
+	}
+
+	caStr, ok := cm.Data[fileName]
+	if !ok {
+		return fmt.Errorf("failed to find certificate %s in configmap %s", fileName, cmName)
+	}
+
+	err = validateCertificateFormat([]byte(caStr))
+	if err != nil {
+		return fmt.Errorf("failed to validate additional CA certificate %s: %v", fileName, err)
```

Comment:
@raptorsun - update this error message to reflect proxy CA certificate

Reply:
message  changed to just `failed to validate certificate`